digiwf-integrations/digiwf-dms-integration/digiwf-dms-integration-core/src/main/java/de/muenchen/oss/digiwf/dms/integration/application/port/in/SearchFileUseCase.java

```diff
@@ -1,5 +1,15 @@
 package de.muenchen.oss.digiwf.dms.integration.application.port.in;
 
 public interface SearchFileUseCase {
-    String searchFile(String searchString, String user);
+
+    /**
+     * Searches for a file. A Search can be refined by an optional 'Fachdatum'/business case.
+     *
+     * @param searchString String to search for
+     * @param user         account name
+     * @param reference    (optional) 'Fachdatum'/business case to refine a search
+     * @param value        (optional) value of 'Fachdatum'/business case
+     * @return File id.
+     */
+    String searchFile(String searchString, String user, String reference, String value);
```

Comment:
Es wird hier nur eine Sachakte zurückgegeben. Wäre eine Liste aller Ergebnisse sinnvoller?

Reply:
Austausch mit @darenegade: Vermutlich hätte in #662 eine Liste durchgereicht werden sollen und nicht nur einer der Treffer.

Wir haben uns entschieden, dafür ein neues Ticket aufzumachen und es zeitnah umzusetzen.